### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by rmg**
Accepts versions as the path or a the v query parameter and optionally
accepts a range as the q query parameter.

### CODE REVIEW

**Comment by rmg** [comment]
@bajtos this is the service that is backing that semver wrapper. It has additional tests :-)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+semver-aas
```
Comment (sam-github) [inline README.md]: "aas"?

Comment (rmg) [inline README.md]: as a service

File: package.json [hunk 2]
```diff
@@ -17,9 +17,13 @@
     "url": "https://github.com/strongloop/satisfies/issues"
   },
   "homepage": "https://github.com/strongloop/satisfies#readme",
```
Comment (bajtos) [inline package.json]: Github URLs seems out of date (semver-aas vs. satisfies).

File: README.md (hunk 1 above)
Comment (bajtos) [inline README.md]: Yeah, I find `-aas` rather cryptic and too close to another word matching `/a.s/` regexp ;) How about "semver-server" or "semver-service"? Just don't abbreviate to [SS](https://en.wikipedia.org/wiki/Schutzstaffel).

**Pull request merged**